ANSWERS.md

```diff
@@ -1,6 +1,10 @@
 ### Questions:
-1. What is the DOM?
-2. What is an event?
-3. What is an event listener?
-4. Why would we convert a NodeList into an Array?
-5. What is a component? 
+What is the DOM? With the DOM, we can manipulate, interact with, change structures, content, styling of web pages. The DOM represents the document as nodes and objects. That way, programming languages can connect to the page.
+
+What is an event? Every interaction a user has with a site.
+
+What is an event listener? A procedure or function in a computer program that waits for an event to occur.
```

Comment:
That is a bit general, in javascript an event listener listen for an event, then calls a callback function!

Reply:
That is true my answer is very general sorry. I know what an eventlistener is I just need to work on my explanation skills.